common/tools/dev-tool/src/commands/run/testVitest.ts

```diff
@@ -93,7 +94,7 @@ export default leafCommand(commandInfo, async (options) => {
       if (options["test-proxy-debug"]) process.env["Logging__LogLevel__Default"] = "Debug";
 
       // restore recordings first in CI to avoid impacting the first playback test
-      if (process.env["BUILD_BUILDNUMBER"]) {
+      if (process.env["BUILD_BUILDNUMBER"] && (await fs.pathExists("assets.json"))) {
```

Comment:
Why not just use `fs.existsSync` instead of the `fs-extra`?

Reply:
Went with the flow of other instances in the project that used fs-extra.
Moved to `fs.existsSync`.